app/assets/stylesheets/modules/home.scss

```diff
@@ -0,0 +1,47 @@
+$good_dog_font: "GoodDog";
```

Comment:
Name of variable `good_dog_font` should be written in all lowercase letters with hyphens instead of underscores